app/controllers/application_controller/ci_processing.rb

```diff
@@ -819,57 +825,18 @@ def process_storage(storages, task)
     end
   end
 
-  def storage_button_operation(method, display_name)
-    storages = []
-    # Either a list or coming from a different controller (eg from host screen, go to its storages)
-    if params.key?(:miq_grid_checks)
-      storages = find_checked_ids_with_rbac(Storage)
-
-      if method == 'scan' && !Storage.batch_operation_supported?('smartstate_analysis', storages)
-        render_flash_not_applicable_to_model(_('Smartstate Analysis'), ui_lookup(:tables => "storage"))
-        return
-      end
-      if storages.empty?
-        add_flash(_("No Datastores were selected for %{task}") % {:task => display_name}, :error)
-      else
-        process_storage(storages, method)
-      end
-
-      if @lastaction == "show_list"
-        show_list unless @explorer
-        @refresh_partial = "layouts/gtl"
-      end
-
-    else # showing 1 storage
-      if params[:id].nil? || !Storage.exists?(params[:id])
-        add_flash(_("Datastore no longer exists"), :error)
-      else
-        storages.push(find_id_with_rbac(Storage, params[:id]))
-        process_storage(storages, method)  unless storages.empty?
-      end
-
-      params[:display] = @display
-      show
-      if ["vms", "hosts"].include?(@display)
-        @refresh_partial = "layouts/gtl"
-      else
-        @refresh_partial = "config"
-      end
-    end
-
-    storages.count
-  end
-
   # Refresh all selected or single displayed Datastore(s)
   def refreshstorage
```

Comment:
I can not find any way to call this action.

@Ladas @lpichler do you know, where I could find such action?
/cc @PanSpagetka

Reply:
@agrare do you know ^?

Reply:
Hm I don't think storage has a refresh_ems method and given it can belong to multiple EMSs I'm not sure how that would even work (refresh all of them?)